content/en/blog/posts/2020-10-21-intro-shipwright-pt1/index.md

```diff
@@ -8,13 +8,14 @@ resources:
 - src: "**.{png}"
   title: "Figure #:counter"
 ---
+*Update 2020-11-25: Added link to Part 2 of this series*
 
 What is Shipwright? Which problems does this project try to solve?
 
 In [Part 1](/blog/2020/10/15/introducing-shipwright-part-1) of this series, we'll look back at the history of delivering software applications,
```

Comment:
Are these `[Part 1](/blog/2020/10/15/introducing-shipwright-part-1)` and `[Part 2](/blog/2020/11/25/introducing-shipwright-part-2)` a Markdown link from this repo? I found they can not be accessed correctly

Reply:
These are local links in the rendered site. You can test it with the Netlify `deploy-preview` details.